client/js/analysis.js

```diff
@@ -1,15 +1,123 @@
-// Listen for on submit click
-let formSubmit = document.getElementById("formSubmit");
-formSubmit.addEventListener("click", fetchAndShowResponse, false);
-
-/**
- * Grabs response from server
- */ 
-async function fetchAndShowResponse() {
-  const response = await fetch('/api/musix');
-  const value = await response.json();
-  const el = document.getElementById(elId); 
-  el.innerText = value;
-  return value;
+window.onload = function() {
+  for (let i = 1; i < 6; i++) {
+    setTimeout(() => {
+      addListElement(i);
+    }, i * 1500);
+  }
+  setTimeout(() => {
+    addFeedbackResult('POSITIVE');
+  }, 9000);
+
+  for (let i = 0; i < 6; i++) {
+    const values = [
+      'TOXICITY',
+      'IDENTITY ATTACK',
+      'INSULT',
+      'PROFANITY',
+      'THREAT',
+      'FLIRTATION',
+    ];
+    addDonutChart(values[i], Math.floor(Math.random() * 61) + 30);
```

Comment:
get rid of magic nums 6, 1600, 61, 30, 9000

Reply:
1500, 9000 aren't magic numbers. They are the number of milliseconds of delay before displaying each object. These must stay for the website to work: https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout

6, 61, and 30 are there so that the code runs and you can see the animations. Wouldn't be able to have the screencast without them. Later today when I put everything together these will be replaced. 

6: The for loop starts at 1 and ends at 6. As you can see in line 5, I'm multiplying that number by 1500 to cause i*1500 ms of delay between each list element. This allows each list to appear staggered. When I put everything together, this will be replaced by something like `commentlist.length`

61 and 30 just set the range of the number `Math.random()` can generate. This generates a random percentage for each donut chart, which allows for the screencast and gives the effect that data is being handled in the donut charts even though it isn't at the moment. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random

Reply:
okay! As long as you replace hard coded nums later today (as you said) that sounds good.

Reply:
For the random data, make it its own line with a comment:
```
// Mock the data with a number between 30% and 60% for now.
const chartFullness = Math.floor(Math.random() * 61 + 30);
addDonutChart(values[i], chartFullness);
```